engines/got/views/splash_screen.cpp

```diff
@@ -0,0 +1,167 @@
+/* ScummVM - Graphic Adventure Engine
+ *
+ * ScummVM is the legal property of its developers, whose names
+ * are too numerous to list here. Please refer to the COPYRIGHT
+ * file distributed with this source distribution.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#include "common/file.h"
+#include "got/views/splash_screen.h"
+#include "got/gfx/palette.h"
+#include "got/vars.h"
+
+namespace Got {
+namespace Views {
+
+#define SPLASH_FRAME_INTERVAL 2
+#define SPLASH_FRAME_COUNT 36
```

Comment:
In v1.0:133, this is 23 (for the Software Creations logo)

Reply:
Okay, I've made the frame count be allowed to be variable. However, the animation of the frames relies on the simplistic assembly instruction interpreter the splash screen implements. So I don't know if the version with that alternate logo will work or not. If it doesn't, point me to a copy of it, and I can look into adding whichever extra few instructions are needed.

Reply:
Here's a copy of all 4 versions I found including the full versions since it's freeware
[GOT.zip](https://github.com/user-attachments/files/18325633/GOT.zip)